### ISSUE TITLE
Add isFollowing in the user properties (Suggestion)

### ISSUE THREAD

**Issue by JaquelineQC** [Androz2091/insta.js#16]
It could be helpful to add isFollowing in the user properties

**Comment by Androz2091** [comment]
isFollowing the client?

**Comment by TommyQC** [comment]
> isFollowing the client?

yes

**Comment by Mr-KayJayDee** [comment]
Or maybe isFollowing(userOne, userTwo) check if the userOne is following userTwo ?

**Comment by TommyQC** [comment]
> Or maybe isFollowing(userOne, userTwo) check if the userOne is following userTwo ?

this could be better use of this function

**Comment by Androz2091** [comment]
Yes I think it's better because to be sure that the list of the followers of a user is up-to-date we need to make a request to Instagram, and it won't work with a getter.
